### DISCUSSION TITLE
If I wanted to enable printing from a sandboxed application?

### DISCUSSION THREAD

**Discussion by NyarlathotepZERO** [sandboxie-plus/Sandboxie#2204] [Q&A Feedback]
How would I go about allowing a sandboxed application to print a document ?

[nevermind, there was a dialogue box that allowed me to bypass the sandbox]

**Comment by bastik-1001** [comment]
Just in case anyone looks for the options to block access to the printer spooler, or how to allow access again.

![Sandboxie printer options](https://user-images.githubusercontent.com/1449429/188175539-31403956-7ab3-461d-88c4-fbbc662df301.png)

Just look under "General Option" and then open the "Access Restrictions".
